website/content/docs/commands/job/start.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: 'Commands: job start'
+description: |
+  The start command is used to start a stopped job.
+---
+
+# Command: job start
+
+The `job start` command is used to start a stopped job.
+
+The start command will use a Consul token with the following preference:
+first the `-consul-token` flag, then the `$CONSUL_HTTP_TOKEN` environment variable.
+Because the consul token used to [run] the targeted job version was not
+persisted, it must be provided to start if the targeted job version includes
+Consul Connect enabled services and the Nomad servers were configured to require
+[consul service identity] authentication.
+
+The Start command will use a Vault token with the following preference:
+first the `-vault-token` flag, then the `$VAULT_TOKEN` environment variable.
+Because the vault token used to [run] the targeted job version was not
+persisted, it must be provided to start if the targeted job includes
+Vault policies and the Nomad servers were configured to require [vault policy]
+authentication.
+
+## Usage
+
+```shell-session
+nomad job start [options] <job>
```

Comment:
```suggestion
nomad job start [options] <job_id>
```

Change to use `<job_id>`, which is the syntax that the [Manage Jobs tutorial](https://developer.hashicorp.com/nomad/tutorials/manage-jobs/jobs-version) uses.